ios/FluentUI.Demo/FluentUI.Demo/Demos/AliasColorTokensDemoController.swift

```diff
@@ -126,14 +126,13 @@ class AliasColorTokensDemoController: DemoTableViewController {
              .presenceDnd,
              .presenceAvailable,
              .presenceOof:
-            return UIColor(dynamicColor: aliasTokens.colors[.foregroundLightStatic])
+            return UIColor(dynamicColor: fluentTheme.color(.foregroundLightStatic))
         }
     }
 
-    private var aliasTokens: AliasTokens {
-        return self.view.fluentTheme.aliasTokens
+    private var fluentTheme: FluentTheme {
+        return self.view.fluentTheme
```

Comment:
Is it worth moving this to an extension on UIViewController?

Reply:
I'd rather not actually, because I don't like the side-effect of calling `view` on a `UIViewController`, since it may load the view if it has not yet been loaded. If a client wants to do that, I want them to do it explicitly.